tests/timeseries/_formula_engine/test_formula_composition.py

```diff
@@ -170,6 +170,27 @@ async def test_formula_composition_missing_bat(self, mocker: MockerFixture) -> N
 
         assert count == 10
 
+    async def test_formula_composition_constant(self, mocker: MockerFixture) -> None:
+        """Test the composition of formulas with constant values."""
+        mockgrid = MockMicrogrid(grid_side_meter=True)
+        await mockgrid.start(mocker)
+
+        logical_meter = microgrid.logical_meter()
+        engine = (logical_meter.grid_power + Power.from_watts(100)).build(
+            "scaled grid_power"
+        )
+        await mockgrid.mock_resampler.send_meter_power([100.0])
+
+        scaled_grid_power = await engine.new_receiver().receive()
+        assert scaled_grid_power.value is not None
+        assert math.isclose(
+            scaled_grid_power.value.as_watts(),
+            200.0,
+        )
+
+        await mockgrid.cleanup()
+        await engine._stop()  # pylint: disable=protected-access
```

Comment:
One last thing,  could you add `await logical_meter.stop()` as well,  so that the `grid_power` engine gets cleaned up as well,  else there will be weird warning messages in test logs, that make it harder to debug actual issues.

Reply:
Thanks! good catch. I wanted to do it and than missed it somehow.